docs: release v1.22 blog post boilerplate (#4013)
aws/copilot-cli: site/content/blogs/release-v122.en.md

---

> title: 'AWS Copilot v1.22: Try out IAM Permission Boundary and more!'

Reviewer: ```suggestion
title: 'AWS Copilot v1.22: Try out IAM Permissions Boundaries and more!'
```

twitter_title: 'AWS Copilot v1.22'
image: ''
image_alt: ''
image_width: '1051'
image_height: '747'
---

> # AWS Copilot v1.22: Try out IAM Permission Boundary and more!

Reviewer: ```suggestion
# AWS Copilot v1.22: Try out IAM Permissions Boundaries and more!
```

- **IAM Role Permissions Boundary**: [See detailed section](#iam-role-permissions-boundary).
- **FIFO SNS/SQS**: [See detailed section](#fifo-snssqs).
- **CloudFront TLS Termination**: You can now use CloudFront to perform faster TLS termination! [See detailed section](#cloudfront-tls-termination).

???+ note "What’s AWS Copilot?"

    The AWS Copilot CLI is a tool for developers to build, release, and operate production ready containerized applications on AWS.
    From getting started, pushing to staging, and releasing to production, Copilot can help manage the entire lifecycle of your application development.
    At the foundation of Copilot is AWS CloudFormation, which enables you to provision infrastructure as code.
    Copilot provides pre-defined CloudFormation templates and user-friendly workflows for different types of micro service architectures,
    enabling you to focus on developing your application, instead of writing deployment scripts.

    See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.

## IAM Role Permissions Boundary

## FIFO SNS/SQS

## CloudFront TLS Termination

> You can now configure env manifest to have CloudFront terminate TLS for your Load Balanced Web Services (LBWS):

Reviewer: ```suggestion
You can now configure your env manifest to have CloudFront terminate TLS for your Load Balanced Web Services (LBWS):
```

```yaml
cdn:
  tls_termination: true
```

> The configuration above uses CloudFront for TLS termination, which means the traffic from `CF → ALB → ECS` will be HTTP only. This brings faster TLS termination and shorter page loading for viewers, since the CloudFront edges are usually geographically closer to the them.

Reviewer: ```suggestion
The configuration above uses CloudFront for TLS termination, which means the traffic from `CF → ALB → ECS` will be HTTP only. This brings faster TLS termination and shorter page loading for viewers, since the CloudFront edges are usually geographically closer to them.
```

> However, if your services have HTTPS enabled (either you have an app domain or imported certificates in the environment), you must turn off ALB http redirect by updating your LBWS manifests.

Reviewer: ```suggestion
However, if your services have HTTPS enabled (you have either an app domain or imported certificates in the environment), you must turn off ALB http redirect by updating your LBWS manifests.
```

```yaml
http:
  disable_redirect: true
```

> And then redeploy the services before using `env deploy` to enable CloudFront TLS termination.

Reviewer: ```suggestion
And then redeploy the services with `svc deploy` before using `env deploy` to enable CloudFront TLS termination.
```

## CloudFront TLS Termination

You can now configure your env manifest to have CloudFront terminate TLS for your Load Balanced Web Services (LBWS):

```yaml
cdn:
  tls_termination: true
```

The configuration above uses CloudFront for TLS termination, which means the traffic from `CF → ALB → ECS` will be HTTP only. This brings faster TLS termination and shorter page loading for viewers, since the CloudFront edges are usually geographically closer to them.

> However, if your services have HTTPS enabled (you have either an app domain or imported certificates in the environment), you must turn off ALB http redirect by updating your LBWS manifests.

Reviewer 2: I don't know if we wrote LBWS anywhere before 😂 
```suggestion
However, if your services have HTTPS enabled (you have either an app domain or imported certificates in the environment), you must turn off ALB http redirect by updating your [Load Balanced Web Service manifests](path/to/manifest link).
```

Author: yeah it's actually here https://github.com/aws/copilot-cli/pull/4013/files#diff-04ecc586aa589f72329c52799921229fc5ff7e898637bf1c965f2ea4a15a7199R41
